Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -295,19 +307,42 @@ using Zdc = Zdcs::iterator;
 
 namespace vzero
 {
-DECLARE_SOA_INDEX_COLUMN(Collision, collision);
+DECLARE_SOA_INDEX_COLUMN_FULL(GlobalBC, globalBC, uint64_t, Triggers, "fGlobalBC");
 DECLARE_SOA_COLUMN(Adc, adc, float[64]);
 DECLARE_SOA_COLUMN(Time, time, float[64]);
 DECLARE_SOA_COLUMN(Width, width, float[64]);
 DECLARE_SOA_COLUMN(BBFlag, bbFlag, uint64_t);
 DECLARE_SOA_COLUMN(BGFlag, bgFlag, uint64_t);
 } // namespace vzero
 
-DECLARE_SOA_TABLE(VZeros, "AOD", "VZERO", vzero::CollisionId,
+DECLARE_SOA_TABLE(VZeros, "AOD", "VZERO", vzero::GlobalBCId,
                   vzero::Adc, vzero::Time, vzero::Width,
                   vzero::BBFlag, vzero::BGFlag);
 using VZero = VZeros::iterator;
 
+namespace fit
+{
+DECLARE_SOA_INDEX_COLUMN_FULL(GlobalBC, globalBC, uint64_t, Triggers, "fGlobalBC");
+DECLARE_SOA_COLUMN(T0Amplitude, t0Amplitude, float[208]);
+DECLARE_SOA_COLUMN(T0TimeA, t0TimeA, float); // TODO do we need also corrected in the AOD?
```

Comment:
Please comment if we need corrected and uncorrected here

Reply:
In practice we have raw times for each V0, AD and T0 cell. So one single float is definitely not enough to store raw time information. And I don't think we need it in AOD. I would treat those single floats for T0A, T0C, V0A, ADA and ADC as calibrated average times. This is what we need for event selection.

Reply:
Resolved.